src/content/docs/fr/tutorial/2-pages/4.mdx

```diff
@@ -0,0 +1,202 @@
+---
+type: tutorial
+title: Stylez votre page À propos
+description: |-
+  Tutoriel : Construisez votre premier blog Astro —
+  Ajoutez une balise de style Astro pour le stylage localisé sur la page
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Spoiler from '~/components/Spoiler.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+Maintenant que vous avez une page "À propos" avec du contenu vous concernant, stylisons-la !
+
+<PreCheck>
+  - Stylez des éléments sur une seule page
+  - Utilisez des variables CSS
+</PreCheck>
+
+
+## Stylez une page individuelle
+
+En utilisant les balises `<style></style>` d'Astro, vous pouvez styliser des éléments sur votre page. Ajouter des **attributs** et des **directives** à ces balises vous offre encore plus de possibilités de stylisation.
+
+1. Copiez le code suivant et collez-le dans `src/pages/about.astro` :
+
+    ```astro title="src/pages/about.astro" ins={6-11}
+    <html lang="fr">
+      <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width" />
+        <title>{pageTitle}</title>
+        <style>
+          h1 {
+            color: purple;
+            font-size: 4rem;
+          }
+        </style>
+      </head> 
+
+    ```
+
+    Vérifiez toutes les trois pages dans l'aperçu de votre navigateur.
+    
+    - De quelle couleur est le titre de la page de :
+
+        - Votre page d'accueil ?  <Spoiler>noir</Spoiler>
+        - Votre page À propos ? <Spoiler>violet</Spoiler>
+        - Votre page de blog ? <Spoiler>noir</Spoiler>
+
+    - La page avec le plus grand texte de titre est ? <Spoiler>Votre page À propos</Spoiler>
+
+    :::tip
+    Si vous ne parvenez pas à déterminer les couleurs visuellement, vous pouvez utiliser les outils de développement de votre navigateur pour inspecter les éléments de titre `<h1>` et vérifier la couleur du texte appliquée.
+    :::
+
+2. Ajoutez le nom de classe `skill` aux éléments `<li>` générés sur votre page À propos, pour que nous puissions les styliser. Votre code devrait ressembler à ceci :
+
+    ```astro title="src/pages/about.astro" 'class="skill"'
+    <p>Mes compétences sont :</p>
+    <ul>
+      {skills.map((skill) => <li class="skill">{skill}</li>)}
+    </ul>
+    ```
+
+3. Ajoutez le code suivant à votre balise de style existante :
+
+    ```astro title="src/pages/about.astro" ins={6-9}
+    <style>
+      h1 {
+        color: purple;
+        font-size: 4rem;
+      }
+      .skill {
+        color: green;
+        font-weight: bold;
+      }
+    </style>
+    ```
+
```

Comment:
From the [english version](https://github.com/withastro/docs/blob/main/src/content/docs/en/tutorial/2-pages/4.mdx#L83), we've 3 lines (CR) here:
```suggestion



```